drush/Commands/SiteInstallCommands.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Drush\Commands;
+
+use Consolidation\AnnotatedCommand\CommandData;
+use Consolidation\AnnotatedCommand\CommandResult;
+use Drupal\cohesion\Drush\DX8CommandHelpers;
+
+/**
+ * Rebuild site studio when site is installed via drush.
+ */
+class SiteInstallCommands extends DrushCommands {
+
+  /**
+   * Execute site studio rebuild after Acquia CMS site install.
+   *
+   * @hook post-command site-install
+   */
+  public function postCommand($result, CommandData $commandData) {
+    $arguments = $commandData->arguments();
+    $moduleHandler = \Drupal::service('module_handler');
+    if (isset($arguments['profile'][0]) && $arguments['profile'][0] == 'acquia_cms' && $moduleHandler->moduleExists('cohesion')) {
+      // Forcefully clear the cache after site is installed otherwise site
+      // studio fails to rebuild.
+      drupal_flush_all_caches();
+      // Below code ensure that drush batch process doesn't hang. Unset all the
+      // ealier created batches so that drush_backend_batch_process() can run
```

Comment:
Minor nit: Typos. "ensures" and "earlier"

Reply:
Other than these minor typos, I think this approach makes sense to me. I tested it and it works. I think bringing this rebuild back into the site-install command will be much more straightforward for people used to that command.

Reply:
thanks, fixed typos